### PR TITLE
Add node-based SVG renderer

### LINKED ISSUE #385: Support SVG rendering on NodeJS

### PR DESCRIPTION
**Pull request by dabbott**
This adds SVG rendering support when running react-sketchapp through node.

This initial implementation uses the `@lona/svg-model` package to simplify and flatten the SVG structure, making it easier to convert to Sketch data types. The trickiest part, the path conversion, is taken from Lona Studio's PDF conversion (PDF and SVG store paths in roughly the same format).

This only handles the basics of the SVG spec:
- The path elements: rect, circle, line, polyline, polygon, path
- Fill and stroke styles
- Custom viewports

Fix #385 

@mathieudutour

### CODE REVIEW

File: src/jsonUtils/nodeImpl/graphics/curvePoint.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import type { Point } from './types';
+
+// SJCurvePoint types copied from sketchapp-json-flow-types (they're not exported)
+type PointString = string;
+type CurveMode = 0 | 1 | 2 | 3 | 4;
+export type SJCurvePoint = {|
```
Comment (mathieudutour) [inline src/jsonUtils/nodeImpl/graphics/curvePoint.js]: I just published sketchapp-json-flow-types@0.3.6 which exposes SJCurvePoint and SJCurveMode

File: src/jsonUtils/nodeImpl/makeSvgLayer.js [hunk 2]
```diff
@@ -1,9 +1,97 @@
...
 
-// TODO:
-export default function makeSvgLayer(layout: LayoutInfo, name: string /* , svg: string */) {
-  const shape = makeRectShapeLayer(0, 0, layout.width, layout.height);
-  shape.name = name;
-  return shape;
+import { makePathsFromCommands, makeLineCapStyle } from './graphics/path';
+import { unionRects, scaleRect, makeBoundingRectFromCommands, resize } from './graphics/rect';
+
+const svgModel = require('@lona/svg-model');
```
Comment (mathieudutour) [inline src/jsonUtils/nodeImpl/makeSvgLayer.js]: let's require it with the weird hack from here: https://github.com/airbnb/react-sketchapp/blob/master/src/jsonUtils/nodeImpl/requireNodobjC.js#L9

File: package.json [hunk 3]
```diff
@@ -40,6 +40,7 @@
     "watch": "npm run build:react -- --watch"
   },
   "dependencies": {
+    "@lona/svg-model": "^1.0.1",
```
Comment (mathieudutour) [inline package.json]: let's put it in the `optionalDependencies` and the `devDependencies`

File: src/jsonUtils/nodeImpl/graphics/curvePoint.js (hunk 1 above)
Comment (dabbott) [reply src/jsonUtils/nodeImpl/graphics/curvePoint.js]: Updated!

File: src/jsonUtils/nodeImpl/makeSvgLayer.js (hunk 2 above)
Comment (dabbott) [reply src/jsonUtils/nodeImpl/makeSvgLayer.js]: cool, seems like it works

**Comment by mathieudutour** [comment]
:tada:

**Pull request merged**